README.md

```diff
@@ -10,6 +12,15 @@ This repository contains both the Dialect rust programs (protocol), in Anchor, a
 
 Currently, the dialect account rent cost is `~0.059 SOL`.
 
+## Table of Contents
```

Comment:
I know a toc is not super common in a readme... but its quite lengthy. imo this increases readbility. An alternative to the toc would be to add a more distinct separation between the top half "Installation" and bottom half "Local Development". Thoughts?

Reply:
This is great for now. Let's get it in and then consider making the distinction better in the future, like you suggest.